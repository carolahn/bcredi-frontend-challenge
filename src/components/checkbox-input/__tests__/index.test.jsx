import React from "react";
import renderer from "react-test-renderer";

import CheckboxInput from "../index";

describe("CheckboxInput render test", () => {
  it("renders correctly CheckboxInput", () => {
    const tree = renderer
      .create(<CheckboxInput values={{}} register={() => {}} errors={{}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
