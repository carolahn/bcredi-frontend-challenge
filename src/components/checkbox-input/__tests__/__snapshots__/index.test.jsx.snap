// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxInput render test renders correctly CheckboxInput 1`] = `
<div
  className="sc-hKgILt eoiDfF"
  style={
    Object {
      "border": "1px solid #e6e6e6",
    }
  }
>
  <input
    className="register__input--checkbox"
    name="agree"
    style={
      Object {
        "border": "1px solid #e6e6e6",
        "color": "#333333",
      }
    }
    type="checkbox"
    width="320px"
  />
  <span
    className="register__checkbox-label"
  >
    Li e estou de acordo com a
     
    <a
      href="#politica-de-privacidade"
      title="Política de privacidade da Bcredi"
    >
      Política de Privacidade
    </a>
     
    e a
     
    <a
      href="#politica-de-uso"
      title="Política de uso da Bcredi"
    >
      Política de Uso de Informações
    </a>
    .
  </span>
</div>
`;
